nes_py/nes_env.py

```diff
@@ -85,7 +95,8 @@ class NESEnv(gym.Env):
 
     # relevant meta-data about the environment
     metadata = {
-        'render.modes': ['rgb_array', 'human'],
+        'render_modes': ['rgb_array', 'human'],
+        'render_fps': 60,
         'video.frames_per_second': 60
```

Comment:
I believe `video.frames_per_second` can be dropped in favor of `render_fps` ([as seen here](https://gymnasium.farama.org/tutorials/gymnasium_basics/environment_creation/#declaration-and-initialization)).
```suggestion
```

Just make sure to update the sole reference I saw to it [here](https://github.com/ItaiBear/nes-py/blob/gymnasium/nes_py/app/play_human.py#L53) to be `target_frame_duration = 1 / env.metadata['render_fps']`.

Thanks for keeping your fork up-to-date. I've been basically maintaining my own fork on the side as well so I can play around with RL and NES games.